plone/app/z3cform/templates/contentbrowser_display.pt

```diff
@@ -0,0 +1,46 @@
+<span class="contentbrowser-widget ${python:normalizeString(view.field.__class__.__name__)}-field"
```

Comment:
block level elements like `div` or `ul` cannot live within inline elements like `span`. Please change to a `div`.

Reply:
oha ... this is a shallow copy of `relateditems_display.pt` where it's also wrong ... I fix it there too